_posts/2015-02-18-ApiBlueprint-button.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Social blueprint
+excerpt: Share your blueprint with others and learn from masters.
+layout: post
+date: 2015-02-18 00:00:00 +0100
+author: vincenzo
+published: true
+commets: true
+---
+
+Recently, we have received a lot of feedback from users that would like to learn more about API Blueprint and how to present stunning and beautiful documentation for their users.
```

Comment:
It reads a little funny to have "from users" and "for their users" in the same sentence. You may be able to remove "for their users" to clear that up.